### DISCUSSION TITLE
Error: Process completed with exit code 2.

### DISCUSSION THREAD

**Discussion by F1xo999** [community/community#56689] [Programming Help] [answered]
### Body

hello i have a error in my code,this is the exact error.
line 3: syntax error near unexpected token `('
Error: Process completed with exit code 2.
I couldnt  help myself and find the cause, it is coming from the main.yml file. the code is for a website and a contact form so that when somebody presses it I will get a notificaton via email, i have set that up but still its not working this is my respitory i could really use some help/guidance https://github.com/F1xo999/F1xo999.github.io.git

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by airtower-luna** [comment, marked as answer]
Your problem seems to be here: https://github.com/F1xo999/F1xo999.github.io/blob/4fec84dc00cdc4b18bc07a62fbb9f38ca789c4bc/.github/workflows/main.yml#L37-L39

The first line of that `run` block is a proper shell command, the third (where the error occurs) and everything after looks like it's supposed to be Javascript, which the shell can't handle. That's why it fails.

You'll need to pass the Javascript part to the interpreter it's meant for, probably by putting it into a separate file and calling the interpreter on that file, or maybe by moving it to a separate step and [setting the `shell` for that step to the interpreter](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell).

**Reply by F1xo999** [reply]
thanks so much
